docs/JAMULUS_PROTOCOL.md

```diff
@@ -0,0 +1,181 @@
+  ACK(REQ_JITT_BUF_SIZE) ---------------->
+      <------------------------------------ ACK(JITT_BUF_SIZE)
+
+      <------------------------------------ REQ_CHANNELS_INFOS (0x1700)
+  CHANNEL_INFOS (0x1900) ----------------> 
+  ACK(REQ_CHANNELS_INFOS) --------------->
+      <------------------------------------ ACK(CHANNEL_INFOS)
+
+(Optional welcome message)
+      <------------------------------------ CHAT_TEXT (0x1200)
+  ACK(CHAT_TEXT) ------------------------>
+
+      <------------------------------------ VERSION_AND_OS (0x1d00)
+  ACK(VERSION_AND_OS) ------------------->
+
+  CHANNEL_INFOS (0x1900) ----------------> 
+      <------------------------------------ ACK(CHANNEL_INFOS)
+
+      <------------------------------------ RECORDER_STATE (0x2100)
+  ACK(RECORDER_STATE) ------------------->   
+
+
+  REQ_CONNECTED_CLIENTS_LIST (0x1000) ---->    
+      <------------------------------------ ACK(REQ_CONNECTED_CLIENTS_LIST)
+
+  REQ_CHANNEL_LEVEL_LIST (0x1c00) -------->    
+
+  JITT_BUF_SIZE (0x0a00) ---------------->    
+      <------------------------------------ JITT_BUF_SIZE (0x0a00)  
+      <------------------------------------ ACK(JITT_BUF_SIZE)
+  ACK(JITT_BUF_SIZE) -------------------->   
+
+      <------------------------------------ CONN_CLIENTS_LIST (0x1800)
+  ACK(CONN_CLIENTS_LIST) ---------------->   
+  
+  NETW_TRANSPORT_PROPS (0x1400) --------->    
+      <------------------------------------ CONN_CLIENTS_LIST (0x1800)
+      <------------------------------------ ACK(NETW_TRANSPORT_PROPS)
+  ACK(CONN_CLIENTS_LIST) ---------------->    
+```
+
+---
+
+General Streaming Messages
+--
+
+During streaming, some control messages are used.  
+Some typical messages could be:
+
+```
+ Client                                     Server
+
+      <------------------------------------ CLM_CHANNEL_LEVEL_LIST (0xf703) 
+
+  CHANNEL_GAIN (0x0d00) ----------------->    
+
+  CLM_PING_MS (0xe903) ------------------>    
+
+      <------------------------------------ ACK(CHANNEL_GAIN)   
+
+      <------------------------------------ CLM_PING_MS (0xe903) 
+
+  MUTE_STATE_CHANGED (0x1f00) ----------->    
+      <------------------------------------ ACK(MUTE_STATE_CHANGED) 
+
+  NETW_TRANSPORT_PROPS (0x1400) --------->    
+      <------------------------------------ ACK(NETW_TRANSPORT_PROPS) - Reset audio packet sequencing on change
+
+  CHANNEL_PAN (0x1e00) ------------------>    
+      <------------------------------------ ACK(CHANNEL_PAN)   
+```
+
+---
+
+Audio Packet Structure
+--
+The OPUS codec is used to compress the audio over the network and the packets are documented [here](https://datatracker.ietf.org/doc/html/rfc6716).
+
+Jamulus uses a custom OPUS encoder / decoder, giving some different frame sizes, but always uses a 48kHz sample rate. OPUS and OPUS64 codecs are the only supported options currently.
+
+The packet size will vary based on:
+* Stereo vs mono
+* Packet size (64/128/256)
```

Comment:
Hm, not sure this is totally accurate.
I think there's three default buffer sizes in the audio logic.
And there is two packet sizes (regular and small) for network buffers, so I would have expected just two values here?

Reply:
I think this line should refer to buffer size:
```suggestion
* Buffer size (64/128/256 samples)
```